lib/order.rb

```diff
@@ -0,0 +1,20 @@
+
+class Order
+  attr_reader :current_order
+
+  def initialize(menu)
+    @current_order = []
+  end
+
+  def select_item(item)
+    if item == 'chips' or item == 'sausage' or item =='fish'
+      @current_order << item
+    else
+      raise 'please select item from menu'
```

Comment:
Style/SignalException: Use `fail` instead of `raise` to signal exceptions. (https://github.com/bbatsov/ruby-style-guide#fail-method)